infer.py

```diff
@@ -89,11 +72,7 @@ def main():
             #                                       img_name[:-4] + "_h.png"))
             # Image.fromarray(f2).save(os.path.join(ckpt_path, exp_name, '%s_%s' % (exp_name, args['snapshot']),
             #                                       img_name[:-4] + "_l.png"))
-            Image.fromarray(f3).save(os.path.join(opt.output_dir, img_name))
+            Image.fromarray(f3).save(os.path.join(output_dir, img_name))
 
         end = time.time()
         print("Average Time Is : {:.2f}".format((end - start) / len(img_list)))
```

Comment:
It seems that we still need an adapter that will also include a unified way to measure time. Let's make an adapter that will include the same steps for all algorithms: reading data, creating a model, preprocessing the data, getting results from the model, postprocessing the data, and saving the results. In this case, we will be able to guarantee that the algorithms were run under the same conditions. This is how it should look like: [torch_adapter](https://github.com/prime-slam/line-detection-association-dockers/blob/549a849ea048d5023790e9286bb05afe2de653d1/detectors/common/adapter/torch_adapter.py), [adapter for algo](https://github.com/prime-slam/line-detection-association-dockers/blob/549a849ea048d5023790e9286bb05afe2de653d1/detectors/F-Clip/adapter.py)